tests/fixed/not_spacing.php

```diff
@@ -11,7 +11,7 @@
 } elseif (! $test === 0) {
     echo 0;
 } else {
-    echo -1;
+    echo - 1;
```

Comment:
This looks like a bug. It's a unary minus, should not be affected by the rule.

Reply:
Please see https://github.com/squizlabs/PHP_CodeSniffer/issues/429.